content/articles/setting-up-accounts-for-clients.markdown

```diff
@@ -0,0 +1,90 @@
+---
+title: Creating accounts for clients
+excerpt: How to create and manage multiple accounts for clients.
+categories:
+- Account
+---
+
+# Creating accounts for clients
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+<note>
+In order to have multiple members on a single account, you must first [be subscribed to an eligible plan](https://dnsimple.com/pricing) for that account.
+</note>
+
+Creating a separate account is a flexible and secure way for agencies and freelancers to share domain management with their clients.
+
+Keep in mind that when you add your client as a user to an account, the new member will have full access rights on every resource attached to the account, including domains, contacts, SSL certificates, etc.
+
+The member will also have full-admin access to the account, including the ability to access and change billing information and account details. The user will retain his login credentials and 2-factor authentication configuration.
+
+<warning>
+Each customer must have a separate account, otherwise if you add more than one customer to the same account each customer will be able to manage other customers' domains.
+</warning>
+
+
+## Creating a separate account for your client
+
+First, you must add another account to your user profile by taking the following steps.
```

Comment:
We should probably link to http://support.dnsimple.com/articles/account-multi/#creating-a-separate-account here.